vector/src/main/java/im/vector/riotx/features/notifications/NotificationDrawerManager.kt

```diff
@@ -499,7 +507,8 @@ class NotificationDrawerManager @Inject constructor(private val context: Context
     companion object {
         private const val SUMMARY_NOTIFICATION_ID = 0
         private const val ROOM_MESSAGES_NOTIFICATION_ID = 1
-        private const val ROOM_EVENT_NOTIFICATION_ID = 2
+        private const val ROOM_INVITATION_NOTIFICATION_ID = 2
+        private const val ROOM_EVENT_NOTIFICATION_ID = 3
```

Comment:
This event was id 2 before. Maybe add the new one as 3 (potential minor migration issue)

Reply:
Yes, good catch, thanks